### PR TITLE
Enable IPFIX for lte setups

### PR DESCRIPTION
**Pull request by koolzz**
**Don't merge need an ovs patch fore  lte**

Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/c/session_manager/LocalEnforcer.cpp [hunk 1]
```diff
@@ -1057,9 +1057,9 @@ bool LocalEnforcer::update_tunnel_ids(
   handle_session_activate_rule_updates(
       imsi, *session, csr, charging_credits_received);
 
+  update_ipfix_flow(imsi, session->get_config(), time_since_epoch);
```
Comment (uri200) [inline lte/gateway/c/session_manager/LocalEnforcer.cpp:1030]: just curious why is this needed now for lte?

File: lte/gateway/c/session_manager/LocalEnforcer.cpp [hunk 2]
```diff
@@ -1057,9 +1057,9 @@ bool LocalEnforcer::update_tunnel_ids(
...
   // if (session_state->get_config().common_context.rat_type() == TGPP_WLAN) {
   if (session->is_radius_cwf_session()) {
-    update_ipfix_flow(imsi, session->get_config(), time_since_epoch);
     if (terminate_on_wallet_exhaust()) {
```
Comment (uri200) [inline lte/gateway/c/session_manager/LocalEnforcer.cpp]: can you use one single if here please?

**Review by uri200 — COMMENTED** [review]
probably you already did it, but just in case: at your vagrant vm `magma/lte/gateway` run
```
make precommit_sm
```
That will run the tests too

File: lte/gateway/c/session_manager/LocalEnforcer.cpp (hunk 1 above)
Comment (koolzz) [reply lte/gateway/c/session_manager/LocalEnforcer.cpp:1030]: some setup requires it

File: lte/gateway/c/session_manager/LocalEnforcer.cpp [hunk 3]
```diff
@@ -1057,12 +1057,10 @@ bool LocalEnforcer::update_tunnel_ids(
   handle_session_activate_rule_updates(
       imsi, *session, csr, charging_credits_received);
 
+  update_ipfix_flow(imsi, session->get_config(), time_since_epoch);
   // if (session_state->get_config().common_context.rat_type() == TGPP_WLAN) {
-  if (session->is_radius_cwf_session()) {
-    update_ipfix_flow(imsi, session->get_config(), time_since_epoch);
-    if (terminate_on_wallet_exhaust()) {
+  if (session->is_radius_cwf_session() && terminate_on_wallet_exhaust()) {
```
Comment (themarwhal) [inline lte/gateway/c/session_manager/LocalEnforcer.cpp]: You can get rid of this too btw `session->is_radius_cwf_session() &&` terminate_on_wallet_exhaust` is an mconfig value that's default true for cwf and default false for lte

**Review by pshelar — APPROVED** [review]

**Review by themarwhal — APPROVED** [review]

**Review by uri200 — APPROVED** [review]

**Comment by koolzz** [comment]
Hey @pshelar I forgot, did you add the patches from this PR to the ovs build that we're currently using?
